File: source/jugglinglab/util/JuggleExceptionInterrupted.java
// JuggleExceptionInterrupted.java
//
// Copyright 2002-2022 Jack Boyce and the Juggling Lab contributors

package jugglinglab.util;


// This is our own exception class, which we use to handle juggling-related
// problems that occur.

public class JuggleExceptionInterrupted extends JuggleExceptionUser {
    public JuggleExceptionInterrupted() {
        super();
    }

    public JuggleExceptionInterrupted(String s) {
        super(s);
    }
}

File: composeApp/src/commonMain/kotlin/jugglinglab/util/JuggleExceptionInterrupted.kt
//
// JuggleExceptionInterrupted.kt
//
// This is our own exception class, which we use to handle juggling-related
// problems that occur.
//
// Copyright 2002-2026 Jack Boyce and the Juggling Lab contributors
//

package jugglinglab.util

class JuggleExceptionInterrupted : JuggleExceptionUser {
    constructor() : super()

    constructor(s: String) : super(s)
}
